interfaces/Portal/src/app/pages/peoples-affected/peoples-affected.page.ts

```diff
@@ -2,22 +2,21 @@ import { Component, OnInit, ViewChild } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import Permission from 'src/app/auth/permission.enum';
 import { ExportType } from 'src/app/models/export-type.model';
-import { Program, ProgramPhase } from 'src/app/models/program.model';
+import { Program, ProgramTab } from 'src/app/models/program.model';
 import { ProgramPeopleAffectedComponent } from 'src/app/program/program-people-affected/program-people-affected.component';
 import { ProgramsServiceApiService } from 'src/app/services/programs-service-api.service';
 
 @Component({
-  selector: 'app-registration-validation',
-  templateUrl: './registration-validation.page.html',
-  styleUrls: ['./registration-validation.page.scss'],
+  selector: 'app-peoples-affected',
+  templateUrl: './peoples-affected.page.html',
+  styleUrls: ['./peoples-affected.page.scss'],
 })
-export class RegistrationValidationPage implements OnInit {
+export class PeoplesAffectedPage implements OnInit {
```

Comment:
"Peoples" is not really a word... 🙈 Also not the plural for "People"...

Reply:
I'd go for `PeopleAffectedPage` (and corresponding file-names).

Or `RegistrationsOverviewPage`, or `AllRegistrationsPage`, etc..

Reply:
Yes, Updated accordingly! Thanks.